test/tests/ver-1ka/comparison_ver-1ka.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
```

Comment:
Look at other .py files for up to date practices:
- Automatic figure generation
- calculate RSMPE